python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -180,6 +180,7 @@ def __init__(self, registry, *args, **kwargs):
         super(RegistryFlusher, self).__init__(*args, **kwargs)
         self.registry = registry
         self._stop = threading.Event()
+        self.logger = getLogger()
```

Comment:
There is, I just wanted to have a separate one for the flusher. Can revert this if there's a strong reason to use the same one as the Registry.